pennylane/measurements/measurements.py

```diff
@@ -171,7 +171,8 @@ def __init__(
         self.id = id
 
         if wires is not None:
-            if len(wires) == 0:
+            print(self.obs, type(self), self.__dict__)
```

Comment:
Please remove this print.

Reply:
This was from another PR; no clue how this sneaked in here 😬 thanks for catching it!